platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -1549,7 +1542,6 @@ public void setupWithOriginalNode(final MerkleNode originalNode) {
             // will NEVER be updated again.
             assert originalMap.isHashed() : "The system should have made sure this was hashed by this point!";
             final VirtualNodeCache<K, V> snapshotCache = originalMap.cache.snapshot();
-            snapshotCache.prepareForFlush();
```

Comment:
`estimatedSize` changes were not reverted

Reply:
Could you clarify, please? `VirtualRootNode.estimatedSize()` is not changed, but it calls methods like `VirtualNodeCache.estimatedDirtyLeavesCount()`, and these methods are reverted, they no longer count filtered leaves